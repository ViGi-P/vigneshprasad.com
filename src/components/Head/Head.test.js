import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { Head } from './'

it('renders header', () => {
  const wrapper = shallow(<Head/>)
  expect(toJson(wrapper)).toMatchSnapshot()
})
