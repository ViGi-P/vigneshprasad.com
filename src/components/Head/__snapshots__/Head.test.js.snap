// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header 1`] = `
<Adapter
  className="Header"
  style={
    Object {
      "background": "url(Background.jpg)",
    }
  }
>
  <Row
    gutter={0}
  >
    <Col
      lg={10}
      md={12}
      sm={12}
      xs={24}
    >
      <Card
        bordered={false}
        className="Main-card"
        cover={
          <img
            alt="DP"
            id="DP"
            src="DP.jpg"
          />
        }
      >
        <Component
          description={
            <div
              className="Main-card-text"
            >
              Software Engineer
            </div>
          }
          title={
            <div
              className="Main-card-text"
            >
              Vignesh Prasad
            </div>
          }
        />
      </Card>
    </Col>
  </Row>
</Adapter>
`;
